packages/jasmine/test/BUILD.bazel

```diff
@@ -3,46 +3,84 @@ load("@npm_bazel_jasmine//:index.bzl", "jasmine_node_test")
 jasmine_node_test(
     name = "underscore_spec_test",
     srcs = ["foo_spec.js"],
-    jasmine = "@npm//jasmine",
-    deps = ["//:jasmine_runner"],
+    # The jasmine label here is specific to local testing since
```

Comment:
nit: lot of duplication - maybe a local macro in this package to make the tests shorter? OTOH it's easier to read without the indirection

at minimum I'd say "see comment above" on the later ones, or maybe extract jasmine as a variable so the comment can go there

Reply:
good idea. extracted duplication to .bzl file